builder/amazon/common/step_pre_validate.go

```diff
@@ -82,22 +84,42 @@ func (s *StepPreValidate) Run(ctx context.Context, state multistep.StateBag) mul
 		return multistep.ActionContinue
 	}
 
+	// Environment variable configurations for prevalidating retries:
+	//
+	// AWS_DESCRIBE_IMAGES_RETRIES - number of times to retry prevalidating the AMI name (default: 5)
+	//
+	// AWS_DESCRIBE_IMAGES_RETRIES_DELAY - delay between retries on looking up the AMI name (default: 10s)
+
 	ec2conn := state.Get("ec2").(*ec2.EC2)
 
 	ui.Say(fmt.Sprintf("Prevalidating AMI Name: %s", s.DestAmiName))
-	resp, err := ec2conn.DescribeImages(&ec2.DescribeImagesInput{
+
+	retries := 5
+	if v, ok := strconv.ParseInt(os.Getenv("AWS_DESCRIBE_IMAGES_RETRIES"), 10, 64); ok != nil {
```

Comment:
just for simplification:

```suggestion
	if v, ok := strconv.Atoi(os.Getenv("AWS_DESCRIBE_IMAGES_RETRIES")); ok != nil {
```